modules/led.sh

```diff
@@ -37,6 +37,17 @@ function led {
     echo "Look at your RPi leds, both leds will be in this pattern... "
     echo "Both LED: 1 sec on; 8 blink; 1 on"
     christmas > "$LOGFILE"
+  elif [ "$color" = "Onam" ]; then
+	checkroot
+	echo "leds are set to Onam mode."
+	echo "Look at your Rpi leds, both leds will be in this pattern..."
+	echo "green LED :5 blink"
+	echo "Both LED : 1 sec off"
+	echo "red LED :5 blink"
+	echo "green LED :5 blink"
+	echo "Both LED : 1 sec"
+	echo "red LED :5 blink"
```

Comment:
We use spaces, not tabs. Just 2 spaces for each level of indentation.

Reply:
Also, a bit pedantic but could you make sure that your spaces and capitalization are consistent?

For example, you have `echo "green LED :5 blink"`, which could be changed to `echo "Green LED: 5 blink"`, and so on.

Also, `Onam` should probably be changed to `onam`.